src/main/java/com/znsio/e2e/tools/Visual.java

```diff
@@ -112,22 +114,38 @@ private com.applitools.eyes.appium.Eyes instantiateAppiumEyes(String driverType,
         appEyes.addProperty(RUN_IN_CI, String.valueOf(getValueFromConfig(RUN_IN_CI)));
         appEyes.addProperty(TARGET_ENVIRONMENT, String.valueOf(getValueFromConfig(TARGET_ENVIRONMENT)));
         appEyes.addProperty("USER_NAME", USER_NAME);
+        String serverUrl = "curl -I --location --request GET '" + getValueFromConfig(APPLITOOLS.SERVER_URL, DEFAULT_APPLITOOLS_SERVER_URL)
```

Comment:
This curl command creation is duplicated in web and app. Why don't you create construct  this command in the method itself?

Reply:
Done